apps/opik-documentation/documentation/fern/docs/faq.mdx

```diff
@@ -74,3 +74,18 @@ For Self-hosted Opik:
 - `url_override` (required): Verify your base URL points to your Opik instance (e.g., `http://your-instance:5173/api`)
 
 You can find your current configuration in the Opik configuration file (`~/.opik.config`) or by checking your environment variables (`OPIK_API_KEY`, `OPIK_WORKSPACE`, `OPIK_URL_OVERRIDE`, `OPIK_PROJECT_NAME`). For more details on configuration, see our [SDK Configuration guide](/docs/opik/tracing/sdk_configuration).
+
+### How can I diagnose issues with Opik?
+
+If you are experiencing any problems using Opik, such as receiving 400 or 500 errors from the backend, or being unable to connect at all, we recommend running the following command in your terminal:
+```bash
+opik healthcheck
+```
+This command will analyze your configuration and backend connectivity, providing useful insights into potential issues. The output will include two key sections:
```

Comment:
> The output will include two key sections:

Let's remove this line. First of all, configuration scan will not always be presented (it will work for localhost and cloud only).
And second - we can add more sections in the future, no need to update the docs every time.